src/javascript/tests.rs

```diff
@@ -228,6 +258,33 @@ function baz(x, y) {
     );
 }
 
+#[test]
+fn importing_a_module() {
+    assert_js!(
+        r#"
+import rocket_ship
+import rocket_ship as foo
+import rocket_ship.{launch as boom_time, fuel}
+// pub fn go() {
+//     rocket_ship.fuel(100)
+//     boom_time()
+// }
+"#,
+        r#""use strict";
+
+import * as rocket_ship from "./rocket_ship.js";
+
+import * as foo from "./rocket_ship.js";
+
+import * as rocket_ship from "./rocket_ship.js";
+const {launch: boom_time, fuel} = rocket_ship;
```

Comment:
decided it was nicer to use the short JS syntax if no alias was given for the name. i.e. render `fuel` rather than `fuel: fuel`. Add's a smidge of complexity to the render code that could be argued is unnecessary.

Reply:
Nice, I like it.

Could we have spaces inside the {}? Could be a good use case for an object helper function that does the formatting and intending for any time we need an object.

Reply:
Is that a standard formatting. I guess there are a lot of opinions in JS but I always write my code blocks with a space and objects without. i.e.
```js
() => { return {x: 1} }
```

Reply:
Where else would the helper be used. I try to not create a new abstraction till I have a couple (ideally 3 places to use it)

Reply:
> Is that a standard formatting. 

Yes I think so. Or at least, it's what prettier, airbnb, and standard do

Reply:
I did my own research and got to the same conclusion.

I've got a wrap_object helper in this PR. https://github.com/gleam-lang/gleam/pull/1071/files#diff-e0fbde2af80ff8ed47119a170f2c53099fef19d489f2b8ccec4ef28105278b3eR443

If we merge that one first I will refactor this PR to use that helper.

Reply:
Updated to use object helper, has spaces.